windows/core/derived/dotnet/core/versions

```diff
@@ -1,23 +1,3 @@
 3.1.101 LTS
-2.1.300-preview2-008530 master
-2.1.101 Current
-2.1.4 Current
-2.1.3 Current
-2.1.2 Current
-2.0.3 Current
-2.0.2 Current
-2.0.0 Current
-1.1.8 LTS
-1.1.7 LTS
-1.1.5 LTS
-1.1.4 LTS
-1.0.4 LTS
-1.0.3 LTS
-1.0.1 LTS
-1.1.8 LTS
-1.1.7 LTS
-1.1.5 LTS
-1.1.4 LTS
-1.0.4 LTS
-1.0.3 LTS
-1.0.1 LTS
\ No newline at end of file
+3.0.102 Current
```

Comment:
I think a the moment we should only support the latest LTS. This can be extended when all the other things are finished.